test/support/utils_sycl.h

```diff
@@ -92,13 +93,31 @@ inline auto default_selector =
 #    endif
 #endif     // ONEDPL_FPGA_DEVICE
 
+#if _ONEDPL_DEBUG_SYCL
+static std::once_flag device_name_in_get_test_queue_logged;
```

Comment:
Let's use `inline` instead of `static` to have only one counter if a test has multiple TUs.

Reply:
Partially fixed.